tox.ini

```diff
@@ -14,3 +14,5 @@ commands =
     python setup.py check
     flake8
     py.test --cov={envsitepackagesdir}/pytest_xpara --cov-append {posargs}
+[run]
+parallel = True
```

Comment:
Could you add a newline at end of file? I forgot to configure `.editorconfig` on this repository.

Reply:
OK